buildSrc/src/main/kotlin/io/spine/internal/dependency/GoogleCloud.kt

```diff
@@ -33,17 +33,11 @@ object GoogleCloud {
     const val core = "com.google.cloud:google-cloud-core:2.3.3"
 
     // https://github.com/googleapis/java-pubsub
-    const val pubSubGrpcApi = "com.google.api.grpc:proto-google-cloud-pubsub-v1:1.115.0"
+    const val pubSubGrpcApi = "com.google.cloud:google-cloud-pubsub:1.115.0"
```

Comment:
Please have a look at `https://mvnrepository.com/artifact/com.google.api.grpc/proto-google-cloud-pubsub-v1/1.97.0`, since it may be the one we actually need.